rust-toolchain.toml

```diff
@@ -0,0 +1,7 @@
+[toolchain]
+channel = "1.60"
```

Comment:
If we're bumping it anyway, why not update it to the latest `rustc` 1.63?
And probably we need to add this to matrix workflow builds (where 1.58 is hardcoded)

Reply:
> If we're bumping it anyway, why not update it to the latest rustc 1.63?

I think I'm the reason for that. I don't use rustup on my laptop, I use the rust toolchain from the Debian repository, and the latest version there, in experimental as of today, is 1.60.

If there's a particular reason, like a feature that is super helpful for us, I'm OK with updating our minimum required toolchain. But otherwise I'd like to stick to something that's more widely available. Admittedly even 1.60 isn't really very widely available yet, debian experimental hardly counts, but I'm ok with that compromise.

It would be good to explain this in a comment here, BTW.

Reply:
Commented.